draft-ietf-quic-tls.md

```diff
@@ -618,11 +618,18 @@ application data, a server MUST treat receiving early data on the TLS connection
 as a connection error of type PROTOCOL_VIOLATION.
 
 
-## Rejecting 0-RTT
+## Accepting and rejecting 0-RTT
 
-A server rejects 0-RTT by rejecting 0-RTT at the TLS layer.  This also prevents
-QUIC from sending 0-RTT data. A server will always reject 0-RTT if it sends a
-TLS HelloRetryRequest.
+A server accepts 0-RTT by sending the EarlyDataIndication in the ServerHello
+(see Section 4.10.2 of {{!TLS13}}). A server that accepts 0-RTT MUST process and
+acknowledge all 0-RTT packets that it receives.
```

Comment:
I am not sure if we need to have this sentence, especially using a normative MUST. IMO, a server should be allowed to drop 0-RTT packets if it needs to (like resource starvation).

Maybe changing the sentence to "Then, the server processes and acknowledges the 0-RTT packets that it receives." would be fine.

Reply:
Makes sense. I applied your suggestion.